metis-authentication/metis-authentication-service/src/test/java/eu/europeana/metis/authentication/service/AuthenticationServiceTest.java

```diff
@@ -1,499 +1,504 @@
-    metisUsers.add(metisUser);
-    metisUsers.add(metisUserAdmin);
-    when(psqlMetisUserDao.getAllMetisUsers()).thenReturn(metisUsers);
-    List<MetisUser> allUsersRetrieved = authenticationService.getAllUsers();
-    assertEquals(metisUsers.size(), allUsersRetrieved.size());
-    assertEquals(metisUsers.stream().map(MetisUserModel::getEmail).collect(Collectors.toSet()),
-            allUsersRetrieved.stream().map(MetisUser::getEmail).collect(Collectors.toSet()));
-  }
-
-  private MetisUserModel registerAndCaptureMetisUser() throws Exception {
-    final ZCRMRecord zcrmRecordContactWithAccountInTheFields = getZCRMRecordContactWithAccountInTheFields();
-
-    when(psqlMetisUserDao.getMetisUserByEmail(anyString())).thenReturn(null);
-    when(zohoAccessClient.getZcrmRecordContactByEmail(anyString()))
-        .thenReturn(zcrmRecordContactWithAccountInTheFields);
-    when(zohoAccessClient.getZcrmRecordOrganizationByName(anyString()))
-        .thenReturn(
-            (ZCRMRecord) zcrmRecordContactWithAccountInTheFields.getFieldValue("Account_Name"));
-    authenticationService.registerUser(EXAMPLE_EMAIL, EXAMPLE_PASSWORD);
-    ArgumentCaptor<MetisUserModel> metisUserArgumentCaptor = ArgumentCaptor.forClass(MetisUserModel.class);
-    verify(psqlMetisUserDao).createMetisUser(metisUserArgumentCaptor.capture());
-    return metisUserArgumentCaptor.getValue();
-  }
-
-  private static ZCRMRecord getZCRMRecordContactWithAccountInTheFieldsNoOrganizationName() {
-    final ZCRMRecord zcrmRecordContact = getZCRMRecordContact();
-    zcrmRecordContact.setFieldValue("Account_Name", getZCRMRecordAccountNoOrganizationName());
-    return zcrmRecordContact;
-  }
-
-  private static ZCRMRecord getZCRMRecordContactWithAccountInTheFields() {
-    final ZCRMRecord zcrmRecordContact = getZCRMRecordContact();
-    zcrmRecordContact.setFieldValue("Account_Name", getZCRMRecordAccount());
-    return zcrmRecordContact;
-  }
-
-  private static ZCRMRecord getZCRMRecordContact() {
-    final ZCRMRecord zcrmRecordContact = new ZCRMRecord("Contacts");
-    zcrmRecordContact.setEntityId(1482250000004168044L);
-    zcrmRecordContact.setFieldValue("Pick_List_3", "EUROPEANA_DATA_OFFICER");
-    zcrmRecordContact.setFieldValue("Metis_user", true);
-
-    return zcrmRecordContact;
-  }
-
-
-  private static ZCRMRecord getZCRMRecordAccount() {
-    final ZCRMRecord zcrmRecordAccount = new ZCRMRecord("Accounts");
-    zcrmRecordAccount.setEntityId(1482250000004168050L);
-    zcrmRecordAccount.setLookupLabel("Europeana Foundation");
-    zcrmRecordAccount.setFieldValue("Organisation_Role2", Collections.singletonList("Aggregator"));
-
-    return zcrmRecordAccount;
-  }
-
-  private static ZCRMRecord getZCRMRecordAccountNoOrganizationName() {
-    final ZCRMRecord zcrmRecordAccount = getZCRMRecordAccount();
-    zcrmRecordAccount.setLookupLabel(null);
-
-    return zcrmRecordAccount;
-  }
-}
\ No newline at end of file
+//package eu.europeana.metis.authentication.service;
```

Comment:
I guess this file should be removed? Or do you intend to fix it at some point? Perhaps it is better to uncomment it and put the `@Ignore` annotation?

Reply:
Darn I knew I will forget about it :D
No totally not, need to fix that..